app/views/users/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container">
+  <div class="user_info">
+    <div class="img">
+      <%= image_tag @user.photo, alt: "User Photo", class: "user-photo"%>
+    </div>
+    <div class="user">
+      <h2><%= @user.name %></h2>
+      <p><strong>Number of posts: </strong><%= @user.posts.count %></p>
+    </div>
+  </div>
+  <div class="user_bio">
+    <p>Bio</p>
+    <p><%= @user.bio %></p>
+  </div>
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div class="post">
+      <h3>Post <%= index+1 %></h3>
+      <p><%= post.title %></p>
+      <p class='count'>Comments: <%= post.comments.count %> | Likes: <%= post.likes.count %></p>
+    </div>
+  <% end %>
+  <%= link_to "See all posts",user_posts_path(@user), class: "button" %> 
```

Comment:
- [ ] The see all posts button should be centered like in the wireframe.

Reply:
Fixed.
![image](https://github.com/Obimbo07/tdn/assets/93598189/9d0f35ff-155d-418f-a73e-f71decb1c427)